src/main/java/aquarium/shop/order/OrderControler.java

```diff
@@ -0,0 +1,37 @@
+package aquarium.shop.order;
+
+
+import aquarium.shop.product.Product;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@RestController
+public class OrderControler {
```

Comment:
Controller should autowire service and proper method should be added in order to communicate with GUI.